examples/shopping-cart/src/components/Cart.js

```diff
@@ -1,37 +1,33 @@
-import React, { Component, PropTypes } from 'react'
+import React, { PropTypes } from 'react'
 import Product from './Product'
 
-export default class Cart extends Component {
-  render() {
-    const { products, total, onCheckoutClicked } = this.props
+const Cart  = ({ products, total, onCheckoutClicked }) => {
+  const hasProducts = products.length > 0
+  const nodes = !hasProducts ?
+    <em>Please add some products to cart.</em> :
+    products.map(product =>
+      <Product
+        title={product.title}
+        price={product.price}
+        quantity={product.quantity}
+        key={product.id}/>
+  )
 
-    const hasProducts = products.length > 0
-    const nodes = !hasProducts ?
-      <em>Please add some products to cart.</em> :
-      products.map(product =>
-        <Product
-          title={product.title}
-          price={product.price}
-          quantity={product.quantity}
-          key={product.id}/>
-    )
-
-    return (
-      <div>
-        <h3>Your Cart</h3>
-        <div>{nodes}</div>
-        <p>Total: &#36;{total}</p>
-        <button onClick={onCheckoutClicked}
-          disabled={hasProducts ? '' : 'disabled'}>
-          Checkout
-        </button>
-      </div>
-    )
-  }
+  return <div>
```

Comment:
I strongly dislike this style. Can we please do a pass in a separate PR that ensures returned multilne JSX is always wrapped in parens? As in:

```js
return (
  <div>
    ...
  </div>
)
```

and

```js
const Stuff = () => (
  <div>
    ...
  </div>
);
```